elements/pfe-sass/mixins/_custom-properties.scss

```diff
@@ -0,0 +1,171 @@
+// New broadcast mixin, for use with content components
+//  1st arg:  Look up values from theme contexts, i.e. "light" or "saturated"
+//  2nd arg:  Optionally specify only one broadcast variable instead of the whole set, i.e. "text"
+//
+//  usage:
+//    :host {
+//      #{pfe-set-broadcast-theme(dark)};
+//    }
+//  result:
+//    :host {
+//      --pfe-broadcasted--color--text:            #{pfe-color(text--on-dark)};
```

Comment:
```suggestion
//      --pfe-broadcasted--color--text: var(--pfe-theme--color--text, #333);
```

Reply:
@castastrophe I put the little code comment updates as one-liners so you could use the commit-suggestion button in github (hopefully saves time)? Let me know if this is more annoying than just adding multiple lines to one suggestion.

Reply:
I'll make the update but they're supposed to be the on-dark versions and the code snippets you added are for the light theme.  I'll make that update this morning.